main/terraform.tfvars

```diff
@@ -0,0 +1 @@
+access_ip = "0.0.0.0/0"
```

Comment:
Please remove this file and migrate the `access_ip` variable to `locals.tf`
Also, instead of `0.0.0.0/0` that is not secure, specify public IP of your ISP